[2024-01-23 05:56:37,291] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: data_cleaning_dag.cleaned_data_message manual__2024-01-23T05:56:35.021176+00:00 [queued]>
[2024-01-23 05:56:37,295] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: data_cleaning_dag.cleaned_data_message manual__2024-01-23T05:56:35.021176+00:00 [queued]>
[2024-01-23 05:56:37,295] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2024-01-23 05:56:37,295] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2024-01-23 05:56:37,295] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2024-01-23 05:56:37,300] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): cleaned_data_message> on 2024-01-23 05:56:35.021176+00:00
[2024-01-23 05:56:37,303] {standard_task_runner.py:52} INFO - Started process 4818 to run task
[2024-01-23 05:56:37,305] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'data_cleaning_dag', 'cleaned_data_message', 'manual__2024-01-23T05:56:35.021176+00:00', '--job-id', '103', '--raw', '--subdir', 'DAGS_FOLDER/data_cleaner/cleaner.py', '--cfg-path', '/tmp/tmpp68kta3y', '--error-file', '/tmp/tmp4ssyaf9x']
[2024-01-23 05:56:37,305] {standard_task_runner.py:80} INFO - Job 103: Subtask cleaned_data_message
[2024-01-23 05:56:37,333] {task_command.py:371} INFO - Running <TaskInstance: data_cleaning_dag.cleaned_data_message manual__2024-01-23T05:56:35.021176+00:00 [running]> on host 0b2a5e032344
[2024-01-23 05:56:37,361] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=deepthisen
AIRFLOW_CTX_DAG_ID=data_cleaning_dag
AIRFLOW_CTX_TASK_ID=cleaned_data_message
AIRFLOW_CTX_EXECUTION_DATE=2024-01-23T05:56:35.021176+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-01-23T05:56:35.021176+00:00
[2024-01-23 05:56:37,361] {logging_mixin.py:115} INFO - Done
[2024-01-23 05:56:37,362] {python.py:173} INFO - Done. Returned value was: None
[2024-01-23 05:56:37,366] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=data_cleaning_dag, task_id=cleaned_data_message, execution_date=20240123T055635, start_date=20240123T055637, end_date=20240123T055637
[2024-01-23 05:56:37,397] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-01-23 05:56:37,414] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
